filebeat/module/elasticsearch/slowlog/_meta/fields.yml

```diff
@@ -10,14 +10,16 @@
   - name: took
     description: "Time it took to execute the query"
     example: "300ms"
-    type: text
+    # Should we extrac that nicer?
+    type: keyword
   - name: types
     description: "Types"
     example: ""
     type: keyword
   - name: stats
     description: "Statistics"
     example: ""
+    # What does this exactly contain, do we need text?
     type: text
```

Comment:
No idea what this field is supposed to contain. Nothing in the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-slowlog.html) about it and the [samples](https://github.com/elastic/beats/blob/master/filebeat/module/elasticsearch/slowlog/test/test.log) we have all have the field as empty.

@pgomulka Since you've been working in making the ES logs structured, do you have any idea what kind of data goes in this `stats` field in the slow log?

Reply:
Should we set index to false for now?

Reply:
it might be  disappointing, but the fields from slow logs we decided to include in a JSON message field.
`stats` field contain group names  https://www.elastic.co/guide/en/elasticsearch/reference/6.6/search.html#stats-groups
sample:
```
{"type": "index_search_slowlog", "timestamp": "2019-01-29T12:49:48,434+0100", "level": "WARN", "component": "i.s.s.query", "cluster.name": "distribution_run", "node.name": "node-0", "cluster.uuid": "oVyn8w_SSOKGqZPQSZfr_A", "node.id": "WocNXE_zSya-68jYBDXf7A",  "message": "[index1][0] took[9.2ms], took_millis[9], total_hits[0 hits], types[], stats[group1,group2], search_type[QUERY_THEN_FETCH], total_shards[1], source[{\"query\":{\"match_all\":{\"boost\":1.0}},\"stats\":[\"group1\",\"group2\"]}], id[], "  }
```

Reply:
I am planning to have a follow up meeting to gather feedback from Beats and monitoring UI. 
refactoring slow logs to have those additional information from message as json fields is in scope too

Reply:
@ycombinator If we can do that, that would be great.

Reply:
Changed to `keyword` field with split processing in https://github.com/elastic/beats/pull/10414.